Log Viewing/ARKLogTableViewController.m

```diff
@@ -71,6 +75,23 @@ - (instancetype)init;
     return [self initWithLogStore:[ARKLogDistributor defaultDistributor].defaultLogStore logFormatter:[ARKDefaultLogFormatter new]];
 }
 
+- (instancetype)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil;
+{
+    return [self init];
+}
+
+- (instancetype)initWithCoder:(NSCoder *)aDecoder;
+{
+    NSAssert(NO, @"Please use a supported initializer.");
+    return [self init];
+}
+
+- (instancetype)initWithStyle:(UITableViewStyle)style;
+{
+    NSAssert(NO, @"Please use a supported initializer.");
```

Comment:
Why not mark this with method with NS_UNAVAILABLE in the header? No need to implement and assert here